docs/design.md

```diff
@@ -260,14 +262,15 @@ runs encounters data that necessitate conflict resolution.
 
 When a transaction restarts, it changes its priority and/or moves its
 timestamp forward depending on data tied to the conflict, and the
-transaction begins anew, updating its intents. Since the set of keys
+transaction begins anew reusing the same tx id. Since the set of keys
 being written change between restarts, a set of keys written during
-prior attempts at the transaction is maintained by the client. As it
-restarts the transaction from the beginning, it removes keys from this
-set as it writes them again. The remaining keys--should the transaction
-run to completion--are crufty write intents which must be deleted
-*before* the transaction commit record’s status is set to COMMITTED.
-Many transactions will have no keys in this set.
+prior attempts at the transaction is maintained by the client as a set of
+dirty keys. As it replays the transaction from the beginning, it 
+removes keys from the dirty set as it writes them again. The remaining
+dirty keys--should the transaction run to completion--are crufty 
+write intents which must be deleted *before* the transaction commit 
+record’s status is set to COMMITTED. Many transactions will end up with
```

Comment:
This paragraph is confusing; what's a crufty write intent? why did the transaction run through to completion without it being evicted?